streams/co.m3u

```diff
@@ -93,13 +93,19 @@ https://live-edge-bhs-1.cdn.enetres.net/184784E1D210401F8041E3E1266822CC021/play
 #EXTINF:-1 tvg-id="" status="online",Amaga TV (720p) (Amagá | Antioquia)
 https://hvtrafico.ddns.net/amagatv/smil:amagatv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",Cinema+ (720p)
-https://hvtrafico.ddns.net/cinema720/cinema720.stream_720p/playlist.m3u8
+https://hvtrafico.ddns.net/cinema720/cinema720.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",Alpavision Canal 2 (Neiva | Huila) (720p) [Not 24/7]
 https://stmv1.voxtvhd.com.br/alpavision/alpavision/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",Canal CNC Montería (720p)
 https://stmv1.voxtvhd.com.br/cncmonteria/cncmonteria/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",Teleamiga (Bogotà | Cundinamarca) (480p) [Not 24/7]
 #EXTVLCOPT:http-referrer=https://player.instantvideocloud.net/
 https://liveingesta118.cdnmedia.tv/teleamigatvlive/smil:dvrlive.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="error",LATAM TV (Bogotà) (540p) [Not 24/7]
-https://iptv-all.lanesh4d0w.repl.co/latam
+#EXTINF:-1 tvg-id="" status="online",LATAM TV (Bogotà) (540p) [Not 24/7]
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://livestream.com/latamtv/envivo
```

Comment:
<img width="748" alt="image" src="https://user-images.githubusercontent.com/7253922/162544294-7e87baee-eac4-4400-b46b-fb47abc1918a.png">

Reply:
it loads out to an not working stream because of a fix not implemented yet

if you want to try : https://q-s-l4n3sh4d0w.onrender.com/iptv-query?streaming-ip=https://livestream.com/latamtv/envivo